modules/core/test/v2/unit/coins/trx.ts

```diff
@@ -0,0 +1,19 @@
+import * as Promise from 'bluebird';
+import * as should from 'should';
+
+const co = Promise.coroutine;
+import { TestBitGo } from '../../../lib/test_bitgo';
+
+describe('TRON:', function() {
+  let bitgo;
+
+  before(function() {
+    bitgo = new TestBitGo({ env: 'mock' });
+    bitgo.initializeTestVars();
+  });
+
+  it('should hoist the coin', function() {
```

Comment:
hoist seems like a strange word, maybe you mean host?

The test is actually testing if we can get an instance of the coin.

Maybe something like this would be better?:
```
it('should instantiate trx coin', () => {
  const basecoin = bitgo.coin('trx');
  basecoin.should.be.an.instanceof(Trx);
});
```

Reply:
done